spatial/r3/vector.go

```diff
@@ -83,6 +83,18 @@ func Cos(p, q Vec) float64 {
 	return Dot(p, q) / (Norm(p) * Norm(q))
 }
 
+// Divergence returns the volume density of the outward flux of a vector
```

Comment:
The current comment unnecessarily focuses on the physical interpretation of the divergence. A user of the function presumably knows what the divergence is. I'd replace it with something like

```
// Divergence returns the divergence of the vector field fn at the point p,
// approximated using central differences with step sizes given in step.
```

Reply:
With reference to https://github.com/gonum/gonum/pull/1816#discussion_r930844923, s/fn // in the comment and change the parameter name to `field`.